### PR TITLE
Prettier console output

### PR DESCRIPTION
**Pull request by mrxiaozhuox**
image
image

### CODE REVIEW

File: src/assets/dioxus.toml [hunk 1]
```diff
@@ -45,10 +47,12 @@ script = []
 binaryen = { wasm_opt = true }
 
 # use sass auto will auto check sass file and build it.
-[application.tools.sass]
+
+
+# [application.tools.sass]
```
Comment (jkelleyrtp) [inline src/assets/dioxus.toml:52]: Is this supposed to be commented out?

File: src/assets/index.html [hunk 2]
```diff
@@ -9,10 +9,9 @@
 </head>
 <body>
   <div id="main"></div>
-  <!-- Note the usage of `type=module` here as this is an ES6 module -->
   <script type="module">
-    import init from "{base_path}/assets/dioxus/{app_name}.js";
-    init("{base_path}/assets/dioxus/{app_name}_bg.wasm").then(wasm => {
+    import init from "/{base_path}/assets/dioxus/{app_name}.js";
+    init("/{base_path}/assets/dioxus/{app_name}_bg.wasm").then(wasm => {
```
Comment (jkelleyrtp) [inline src/assets/index.html:14]: Does this change the behavior at all for sub routes?

**Review by jkelleyrtp — APPROVED** [review]
Looks good but two small questions

File: src/assets/dioxus.toml (hunk 1 above)
Comment (mrxiaozhuox) [reply src/assets/dioxus.toml:52]: I don't think sass support should be turned on by default,how do you think?

File: src/assets/index.html (hunk 2 above)
Comment (mrxiaozhuox) [reply src/assets/index.html:14]: this change will fixed sub-routes problem

**Pull request merged**